app/src/main/java/app/revanced/integrations/youtube/swipecontrols/SwipeControlsHostActivity.kt

```diff
@@ -198,7 +200,7 @@ class SwipeControlsHostActivity : Activity() {
      */
     private fun createScreenController() =
         if (config.enableBrightnessControl) {
-            ScreenBrightnessController(this)
+            ScreenBrightnessController(this, config)
```

Comment:
The current activity is injected into the ScreenBrightnessController and the config is a property of it. Change the constructor of ScreenBrightnessController to inject the same type as its done in ScreenBrightnessController and access the config through the property just like ScreenBrightnessController instead of adding a second config parameter.